Lib/sysconfig.py

```diff
@@ -112,6 +112,11 @@ def _safe_realpath(path):
 # PC/VS7.1
 if os.name == "nt" and "\\pc\\v" in _PROJECT_BASE[-10:].lower():
     _PROJECT_BASE = _safe_realpath(os.path.join(_PROJECT_BASE, pardir, pardir))
+# PC/VS9.0/amd64
+if (os.name == "nt" 
+   and os.path.basename(os.path.dirname(os.path.dirname(_PROJECT_BASE))).lower() == "pc" 
+   and os.path.basename(os.path.dirname(_PROJECT_BASE)).lower().startswith("vs")):
```

Comment:
I'm not sure about the .startswith("vs") part, maybe I should replace it with == "vs9.0". What do you think?